recommender/recommender.py

```diff
@@ -150,8 +203,10 @@ def get_user_is_staff(self):
         being defined. However, It's the only way to get the data right now.
         TODO: Should be proper handled in future
         """
+        if "workbench" in str(type(self.runtime)):
+            return True
```

Comment:
XBlocks doesn't have a user service. One is being developed. This is a bit of a short-term hackaround until it is in place. We have appropriate doc strings and TODOs around this right now.